mtp_cashbook/templates/disbursements/start.html

```diff
@@ -11,23 +11,34 @@
   <div class="mtp-disbursement-form-preview grid-row">
     <div class="column-two-thirds">
       <header>
-        <h1 class="heading-xlarge">{% trans 'Digital disbursements' %}</h1>
+        <h1 class="heading-xlarge">{% trans 'Digital disbursements tool' %}</h1>
       </header>
+      <h3><strong>{% trans 'How does it work' %}</strong></h3>
```

Comment:
with headings, you need to use one of the heading classes as in 2 lines up from here. i'd try `heading-medium` or a smaller one if there is one